app/lib/message_list/text_message_tile.dart

```diff
@@ -143,16 +145,19 @@ class TextMessageTile extends StatelessWidget {
             borderRadius: BorderRadius.circular(7),
             color: isSender ? colorDMB : colorDMBSuperLight,
           ),
-          child: RichText(
-            text: buildTextSpanFromText(
-                ["@Alice", "@Bob", "@Carol", "@Dave", "@Eve"],
-                text,
-                messageTextStyle(context, isSender),
-                HostWidget.richText),
-            selectionRegistrar: SelectionContainer.maybeOf(context),
-            selectionColor: Colors.blue.withValues(alpha: 0.3),
-            textWidthBasis: TextWidthBasis.longestLine,
+          child: DefaultTextStyle.merge(
+            style: messageTextStyle(context, isSender),
+            child: Column(
+              children: messageContent.content
+                  .map((inner) => buildBlockElement(inner.element, isSender))
+                  .toList(),
+            ),
           ),
+          // child: RichText(
+          //   selectionRegistrar: SelectionContainer.maybeOf(context),
+          //   selectionColor: Colors.blue.withValues(alpha: 0.3),
+          //   textWidthBasis: TextWidthBasis.longestLine,
+          // ),
```

Comment:
I disabled this for now. Should I add it back in?

Reply:
If you mean that "@alice", etc. get special treatment, then yes, that's okay to disable. I think that was just an experiment to begin with.

Reply:
I meant the selectionRegistrar code

Reply:
My flutter/dart isn't great. This was enabling rich text wasn't it? If so then I don't think we need it anymore after your PR.

Reply:
I'll remove it